14.0/base_requirements.txt

```
@@ -47,7 +47,7 @@ python-dateutil==2.7.3
 pytz==2019.1
 pyusb==1.0.2
 qrcode==6.1
-reportlab==3.5.13; python_version < '3.8'
+reportlab==3.5.54; python_version < '3.8'
```

Usage is to keep a reference to the official dep version in a comment:
```suggestion
reportlab==3.5.54; python_version < '3.8'  # official 3.5.13
```

---

thanks for your review I updated the pr